src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -543,6 +543,14 @@
 
     public static final Key<Value<Vector3d>> LEFT_ARM_ROTATION = KeyFactory.fake("LEFT_ARM_ROTATION");
 
+    /**
+     * Represents the {@link Key} for representing the dominant {@link HandType}
+     * of a {@link Living} entity.
```

Comment:
@kashike Not at all. `Player`s can have their handedness changed, which leaves the only other living type to be `Agent`s (which the implementation follows fyi).